src/commands/contract/deploy.ts

```diff
@@ -97,6 +97,22 @@ export class DeployContract extends SwankyCommand<typeof DeployContract> {
       return new ChainAccount(mnemonic);
     }, "Initialising")) as ChainAccount;
 
+    const buildMode = await contract.getBuildMode();
+    if(buildMode !== 'Release') {
```

Comment:
We can extend this check also making sure the contract was build in `--verifiable` mode. This feature is coming in this PR #194. You can verify the image version in the metadata file making sure it is: `paritytech/contracts-verifiable`